score-vmware/src/main/java/io/cloudslang/content/vmware/constants/ErrorMessages.java

```diff
@@ -0,0 +1,30 @@
+package io.cloudslang.content.vmware.constants;
+
+/**
+ * Created by Mihai Tusa.
+ * 10/20/2015.
+ */
+public class ErrorMessages {
```

Comment:
maybe use enum instead for the constants as these classes tend to get big in size over time